recipes/snippy/3.0/meta.yaml

```diff
@@ -13,7 +13,7 @@ build:
 
 requirements:
   build:
-    - perl-threaded
+    - perl
```

Comment:
needs build bump

Reply:
I decided against this, it would not change anything, but with the next version bump people will have it easier.

Reply:
fine by me